src/library/librarycontrol.cpp

```diff
@@ -172,6 +173,16 @@ LibraryControl::LibraryControl(Library* pLibrary)
             });
 #endif
 
+    // Pure trigger control. Alternative for signal/slot since widgets that want
+    // to call refocusPrevLibraryWidget() are cumbersome to connect to.
+    // This CO is never actually set or read so the value just needs to be not 0
+    m_pRefocusPrevWidgetCO = std::make_unique<ControlPushButton>(
+            ConfigKey("[Library]", "refocus_prev_widget"));
+#if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
```

Comment:
What is the reason to not connect it in Qt 6?

Reply:
QML. No legacy library widgets, so it's guarded like the other functions that work with these widgets.